scipy/ndimage/tests/test_filters.py

```diff
@@ -533,6 +534,7 @@ def test_correlate22(self, dtype_array, dtype_output, xp):
         assert_array_almost_equal(output, expected)
 
     @skip_xp_backends("jax.numpy", reason="output array is read-only.")
+    @skip_xp_backends("dask.array", reason="output array is read-only.")
```

Comment:
dask arrays are never read-only. Please review this and all lines below.

Reply:
It's the same/a similar issue to what I was trying to solve with the contiguous copy above.

The output from ``da.zeros`` is not writeable.
```
>>> a = np.asarray(da.zeros(10))
>>> a.flags
  C_CONTIGUOUS : False
  F_CONTIGUOUS : False
  OWNDATA : False
  WRITEABLE : False
  ALIGNED : True
  WRITEBACKIFCOPY : False
```